test/Serialization/upgrades/file_version<=0.11.1.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Would it make sense to also keep a version of file but for 0.11.1 in the repo?

Reply:
you mean 0.11.2? which would be the current latest format since i assume this new format only gets released with the next version bump. I would include that file when we have an upgrade script from 0.11.2 to the next upgrade

Reply:
Sorry. Yeah, I meant to keep both a file in the "old" format and in the "new" format (or later: keep a version of the "same" (?) file in all format versions), so that we can verify the chain of upgrades works as intended. At the same time it makes it easy to "visualize" what really changed

Reply:
I am thinking that this might not work as expected? since we might have a change that doesn't affect the file? or that there maybe be certain upgrades that wouldn't affect the file. I guess we could try and include a file that has every possible type that can be serialized? but then it would be pretty big?